docs/SIL.rst

```diff
@@ -5707,6 +5707,49 @@ Extracts the original function or a transpose function from the given
 ``[original]`` or ``[transpose]``.
 
 
+differentiability_witness_function
+``````````````````````````````````
+
+::
+
+  sil-instruction ::= 'differentiability_witness_function'
+                      sil-differentiability-witness-function-kind
```

Comment:
```suggestion
                      '[' sil-differentiability-witness-function-kind ']'
```

Reply:
Done in https://github.com/apple/swift/pull/27719/commits/cb30eb756b96f76879e95a6a1a9bb49e3111e0d9.